vm/src/vm.rs

```diff
@@ -198,13 +198,27 @@ impl VirtualMachine {
             vm.get_none(),
         );
 
-        builtins::make_module(&vm, builtins.clone());
-        sysmodule::make_module(&vm, sysmod, builtins);
+        vm
+    }
+
+    pub fn initialize(&mut self, external: bool) -> &Self {
```

Comment:
I think this could return a `PyResult<()>`, since we already have enough of the VM to be able to print errors, and it is helpful occasionally to see an error that came out of `importlib` initialization or similar. Also, does it need to be `&mut self` instead of `&self`?